src/wwwroot/template-editor/adapterService.js

```diff
@@ -415,6 +416,253 @@
       return $q.resolve({});
     }
 
+    /**
+     * Overriding of DropZone's uploadFiles function
+     * 
+     * See https://github.com/MakingSense/MSEditor/pull/1259
+     * 
+     * @param {File[]} files 
+     */
+    function overrideUploadFiles(files) {
+      traceAdapterCall(overrideUploadFiles, arguments);
+
+      var file, formData, handleError, headerName, headerValue, headers, i, input, inputName, inputType, key, method, option, progressObj, response, updateProgress, url, value, xhr, _i, _j, _k, _l, _len, _len1, _len2, _len3, _m, _ref, _ref1, _ref2, _ref3, _ref4, _ref5;
+
+      // DropZone's original code:
+      // method = resolveOption(this.options.method, files);
+      method = 'POST';
+
+      // DropZone's original code:
+      // url = resolveOption(this.options.url, files);
+      url = 'images';
```

Comment:
This could be targeting a fake image on CDN. What do you think?

Reply:
This is the URL for upload the image, and also we are not uploading it thanks of our XMLHttpRequest double so I do not see any advantage on doing that.

Reply:
Oh I see now, thanks andres :)